migrations/versions/b243cdc08370_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: b243cdc08370
+Revises: ('dd7968d8d413', '49f3a33f5437')
+Create Date: 2018-06-09 03:22:33.135172
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = 'b243cdc08370'
+down_revision = ('dd7968d8d413', '49f3a33f5437')
```

Comment:
This whole migration file is redundant and you can simply update the migration file ```dd7968d8d413.py``` to point to the most updated head. 
To find the most updated migration file use ```python manage.py db history```.

Reply:
python manage.py db history gave
which is pointing to this file itself.
```
dd7968d8d413, 49f3a33f5437 -> b243cdc08370 (head) (mergepoint), empty message
49f3a33f5437 -> dd7968d8d413, empty message
```